html_css/intermediate_html/Introduction.md

```diff
@@ -1,14 +1,14 @@
 ### Introduction
 
-This is the first of the two part course where we dig deeper into HTML and CSS.
+This is the first of the two-part course where we dig deeper into HTML and CSS.
 
-Our foundations lessons, by design, only scratched the surface of what's possible. We wanted to give you all of the bare necessities for getting started so you could move on and be somewhat productive as quickly as possible, but now it's time to slow down and dig in!
+Our foundations lessons, by design, only scratched the surface of what's possible. We wanted to give you all of the bare necessities for getting started, so you could move on and be somewhat productive as quickly as possible, but now it's time to slow down and dig in!
 
-You have probably realized by now that there are many more HTML elements than we mentioned in our foundations content. Throughout this course we'll be hitting the rest of the important elements such as forms and tables.
+You have probably realized by now that we mentioned many more HTML elements in our foundation's content. Throughout this course, we’ll hit the rest of the important elements, such as forms and tables.
```

Comment:
This changes the meaning of the sentence. We're saying there are more elements that we *haven't* covered yet. Your sentence implies that we covered more elements in the foundations lesson, which is not the case.